admin-console/src/main/webapp/src/services/SystemBehaviorGroups/GetBehaviorGroups.tsx

```diff
@@ -0,0 +1,46 @@
+import { validatedResponse, validationResponseTransformer } from 'openapi2typescript';
+import { useMemo } from 'react';
+import { useQuery } from 'react-fetching-library';
+
+import { Operations } from '../../generated/OpenapiInternal';
+import { BehaviorGroup } from '../../types/Notifications';
+
+const validateResponse = validationResponseTransformer(
+    (payload: Operations.InternalServiceGetDefaultBehaviorGroups.Payload) => {
+        if (payload.status === 200) {
+            const systemBehaviorGroups: ReadonlyArray<BehaviorGroup> = payload.value.map(value => ({
+                bundleId: value.bundle_id,
+                displayName: value.display_name,
+                actions: value.actions,
+                id: value.id
+            }));
+
+            return validatedResponse(
+                'BehaviorGroups',
+                200,
+                systemBehaviorGroups,
+                payload.errors
+            );
+        }
+
+        return payload;
+    }
+);
+
+export const useSystemBehaviorGroups = () => {
+    const query = useQuery(Operations.InternalServiceGetDefaultBehaviorGroups.actionCreator());
```

Comment:
I guess the reason of why it's appearing on all the bundles is because we are fetching all the system behavior groups - does this action accepts a bundleId ? If not we might need to filter it.